src/main/java/com/google/sps/utils/UserUtils.java

```diff
@@ -62,4 +66,54 @@ public Entity getEntityFromDatastore(String entityName, String entityPropertyTit
     Entity entity = resultsList.get(0);
     return entity;
   }
+
+  /**
+  * Adds a game id to a user's list of games
+  */
+  public static boolean addGameToUser(Entity userEntity, DatastoreService datastore, String gameId) {
+ 
+    if(gameId == ""){
+        log.severe("no gameId");
+        return false;
+    }
+
+    if(userEntity == null || datastore == null){
+        log.severe("null datastore or user entity");
+        return false;
+    }
+ 
+    ArrayList<String> games = (ArrayList<String>) userEntity.getProperty("games");
+ 
+    if(games == null){
+        log.severe("no game list");
+        return false;
+    }
+ 
+    games.add(gameId);
+    userEntity.setProperty("games", games);
+    datastore.put(userEntity);
+ 
+    return true;
+  }
+
+  /**
+  * adds a photo to the user entity
+  */
+  public static boolean addBlobKey(String blobKey, Entity userEntity, DatastoreService datastore) {
```

Comment:
Similar comments about logging to those in the method above.

Reply:
Sounds good, I made similar changes.